storage/mysql/map_storage_test.go

```diff
@@ -167,28 +170,24 @@ func TestMapRootUpdate(t *testing.T) {
 				Signature:      &spb.DigitallySigned{Signature: []byte("notempty")},
 			}},
 		{
-			desc: "Root with default MapperMetadata",
+			desc: "Root with default (empty) MapperMetadata",
 			root: trillian.SignedMapRoot{
 				MapId:          mapID,
 				TimestampNanos: 98768,
 				MapRevision:    7,
 				RootHash:       []byte(dummyHash),
 				Signature:      &spb.DigitallySigned{Signature: []byte("notempty")},
-				Metadata: &trillian.MapperMetadata{
-					HighestFullyCompletedSeq: 0,
-				},
+				Metadata: &any.Any{},
```

Comment:
I find an empty Any a bit unusual. I would expect it to either be nil or be "unmarshable".

Reply:
I don't think this is a realistic test case, but this test is checking round-trips.  Even if unrealistic this structure should still be round-trippable.  I can remove it if you think it's pointless?
The nil case is the one above.
Perhaps I should devise a test where the Any contains unmarshalable data.